src/config.rs

```diff
@@ -166,6 +167,29 @@ impl IndexMut<usize> for ColorList {
     }
 }
 
+#[derive(Debug, Deserialize)]
+pub struct Shell<'a> {
+    program: Cow<'a, str>,
+    args: Vec<String>,
```

Comment:
no worries, how come? and why doesn't the compiler complain/how do I test that it works?